packages/dart_frog_cli/test/src/commands/dev_server_runner/dev_server_runner_test.dart

```diff
@@ -180,11 +183,41 @@ void main() {
         );
       });
 
+      test('custom host ip', () async {
+        devServerRunner = DevServerRunner(
+          logger: logger,
+          host: '0.0.0.0',
+          port: port,
+          devServerBundleGenerator: generator,
+          dartVmServicePort: dartVmServicePort,
+          workingDirectory: Directory.current,
+          directoryWatcher: (_) => directoryWatcher,
+          generatorTarget: (_, {createFile, logger}) => generatorTarget,
+          isWindows: isWindows,
+          startProcess: (
+            String executable,
+            List<String> arguments, {
+            bool runInShell = false,
+          }) async {
+            return process;
+          },
+          sigint: sigint,
+          runProcess: (_, __) async => processResult,
+        );
+
+        await expectLater(devServerRunner.start(), completes);
+
+        expect(devServerRunner.isWatching, isTrue);
+        expect(devServerRunner.isServerRunning, isTrue);
+        expect(devServerRunner.isCompleted, isFalse);
+      });
```

Comment:
Not enough assertion for the tested use case. See the immediately following test, where the custom port number is verified.